handler.js

```diff
@@ -0,0 +1,43 @@
+/* handler.js
+ * Manages tests running in parallel
+ */
+var child_process = require('child_process');
+var configure = require('./configure.js');
+
+exports.startTest = function(name_, command, directory) {
+    var process = child_process.exec(command, { cwd : directory });
+    var container = { name : name_,
+                      test: process };
+
+    process.stdout.on('data', function (data) {
+          console.log('stdout: ' + data);
+    });
+
+
+    process.on('close', function(container) {
+        var index = tests.indexOf(container);
+        tests.splice( index, 1 );
+        exports.updateTests(configure.getState());
+    });
+
+    tests.push(container);
+
+    return process;
+}
+
+exports.killTest = function(test) {
+    test.kill('SIGINT');
```

Comment:
In the future we may want to SIGKILL if the test does not respond for too much time.

(Wondering whether SIGINT and SIGKILL are portable in Node)